### PR TITLE
Aqcu 1195

### PR DESCRIPTION
**Pull request by lindsayplatt**
Just some mismatched parentheses (+ some readability changes)

### CODE REVIEW

File: R/timeseriessummary-data.R [hunk 1]
```diff
@@ -630,12 +630,14 @@ formatCorrectionsParamUSGSMultiPoint <- function(startShiftPoints, endShiftPoint
   if (!isEmptyOrBlank(startShiftPoints) && !isEmptyOrBlank(endShiftPoints) && !isEmptyOrBlank(usgsType)) {
     if (all(c("Value","Offset") %in% names(startShiftPoints))) {
       for (i in 1:nrow(startShiftPoints)) {
-        formattedParameters <- paste0(formattedParameters, "Start shift points value ", startShiftPoints[['Value']][[i]], ", offset ", round(as.numeric(startShiftPoints[['Offset']][[i]],3)), ". ")
+        formattedParameters <- paste0(formattedParameters, "Start shift points value ", startShiftPoints[['Value']][[i]], ", offset ", 
+                                      round(as.numeric(startShiftPoints[['Offset']][[i]]), 3), ". ")
```
Comment (lindsayplatt) [inline R/timeseriessummary-data.R:634]: this was the only one that had the parentheses issue - the rest are just moving the code onto a new line so it's not quite so long

**Comment by mhines-usgs** [comment]
This works fine for me locally. I'm not sure why the build is failing here?

**Comment by mhines-usgs** [comment]
lol, that was weird @lindsaycarr !

**Review by mhines-usgs — APPROVED** [review]
After Travis stopped stumbling, I think we're good. Thanks!

**Pull request merged**